tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py

```diff
@@ -150,6 +150,37 @@ def pods(self, multi_pvc_factory, dc_pod_factory):
             )
         return pod_objs
 
+    @pytest.fixture()
+    def delete_configmap(self):
```

Comment:
Can we rename the function delete_and_recreate_configmap() something like that , since you are recreating configmap in this function wdyt?

And also can we move this function under monitoring.py file?

Reply:
I agree:
- renaming would make the purpose of the function immediately more clear
- moving this function to [monitoring.py](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/monitoring.py) could make sense (this module contains code handling monitoring reconfiguration for OCS)

But the most important detail here is the reason why we are dong this before the test. What will this achieve? What would a failure of this procedure mean? It would be useful to noted this down in docstring or a comment in the test code.